package.json

```diff
@@ -48,10 +48,10 @@
     "@babel/preset-env": "^7.11.0",
     "@babel/traverse": "^7.11.5",
     "@rollup/plugin-babel": "^5.1.0",
-    "@rollup/plugin-commonjs": "^11.0.0",
-    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-commonjs": "^15.1.0",
```

Comment:
See https://github.com/formium/tsdx/pull/727#issuecomment-695894307 : upgrading `commonjs` apparently required Rollup 2 without any note in the Changelog